CDE_schema.rnc

```diff
@@ -57,11 +86,11 @@ org =
 index_codes =
   element index_codes {
     element index_code {
-      attribute scheme { text },
-      attribute code { text },
-      attribute display_name { text }?,
-      attribute url { xsd:anyURI }?
-    }*
+      attribute system { xsd:Name },
+      attribute code { xsd:normalizedString },
+      attribute href { xsd:anyURI }?,
+      attribute display { text }?
```

Comment:
If we're mirroring FHIR - https://www.hl7.org/fhir/datatypes.html#Coding
`
attribute system { xsd:anyURI },
attribute code { text },
attribute display { text },
`

They don't include HREF, and I don't think we need to either.  They do include version - which I'm +/- on including.

Reply:
I'm also meh on including the version.
I think allowing inclusion of the href (it's optional) would be useful, though.

Reply:
Ok:

Let's leave out version.
Let's update system to be a URI
and include HREF as an optional element

Can you submit another commit?